specification/containerservice/resource-manager/Microsoft.ContainerService/stable/2021-05-01/managedClusters.json

```diff
@@ -3171,45 +2998,45 @@
         },
         "kubernetesVersion": {
           "type": "string",
-          "description": "Version of Kubernetes specified when creating the managed cluster."
+          "description": "The version of Kubernetes specified when creating the managed cluster."
```

Comment:
Can this _only_ be specified when creating the cluster? It can't be changed later?

Same question on other fields below that say: "When creating the managed cluster"

If it is true that this cannot be changed, we should probably change this language to say that clearly... something like:
`The version of Kubernetes to use in the cluster. To change this after the cluster has been created, see the Upgrade operation`. Or for stuff like DNS prefix, just say: `This cannot be changed after the cluster has been created`